Description: Loosen expected test case in chk_fhan16
 Upstream says it is only a sorting issue and not an important test failure.
Author: Ximin Luo <infinity0@debian.org>
Bug: http://xcas.e.ujf-grenoble.fr/XCAS/viewtopic.php?p=8965#p8965
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/check/chk_fhan16
+++ b/check/chk_fhan16
@@ -1,4 +1,4 @@
 #! /bin/sh
 unset LANG
 ../src/icas TP16-sol.cas > TP16.tst
-diff TP16.tst TP16-sol.cas.out1
+diff TP16.tst TP16-sol.cas.out1 || diff TP16.tst TP16-sol.cas.out2
--- a/check/chk_integrate
+++ b/check/chk_integrate
@@ -1,4 +1,4 @@
 #! /bin/sh
 export LANG=en_US.UTF8
 ../src/icas testintegrate > integrate.tst
-diff integrate.tst integrate.out
+diff integrate.tst integrate.out || diff integrate.tst integrate.out2
